programmers/난이도별/level01.문자열_다루기_기본/HyeonJeong.py

```diff
@@ -0,0 +1,9 @@
+def solution(s):
+    if len(s) == 4 or 6 :
```

Comment:
```suggestion
    if len(s) != 4 and 6 :
        return False
```
여기서 바로 리턴을 해주는게 좋아요!